Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:3.9-slim
```

Comment:
describes the performance manager image that we'll deploy to ECS. we use this to test out locally.

it will pull in the performance manager files and install dependencies.

Reply:
Would it be worth using an `alpine` or `debian-slim` image in combination with `asdf` and `poetry` give us a little more flexibility/control in terms of install configuration in the future? 

I think the `python-3.9-slim` is already based on a debian bullseye image, so this might just be a matter of semantics.